### PR TITLE
db migration in prep for rethinkdb migration

### PR DESCRIPTION
**Pull request by schuyler1d**
This is step 1 of many.  I'd like to make this as incremental as possible.  See docs/DB_MIGRATION.md for full plan.

### CODE REVIEW

File: src/server/api/schema.js [hunk 1]
```diff
@@ -366,9 +366,8 @@ const rootMutations = {
       await Organization
         .get(organizationId)
         .update({
-          texting_hours_settings: {
-            permitted_hours: [textingHoursStart, textingHoursEnd]
```
Comment (shakalee14) [inline src/server/api/schema.js:370]: this is the array field you are talking about @schuyler1d ?

File: docs/DB_MIGRATION.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Custom model options incompatible with RDBMS
+
+thinky.type
+
+###  .object() # mostly for createModel
+
+ - [ ] campaign-contact.js: custom_fields
+          current features can just be json -- might be worth allowing jsonb in postgres
+ - [ ] job-request.js: payload
```
Comment (shakalee14) [inline docs/DB_MIGRATION.md:23]: agree

File: docs/DB_MIGRATION.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          pending-message-part.js: service_message
+          see array of same kind of stuff.
+          maybe all of this should be in redis instead?
+
+### .array 
+
+ - [X] organization.js
+          for permitted_hours, # actually just two values [start, end]
+          features: just has 'one feature' = 'threeClick'
+             let's make it a string since it's tested vs. filtered
```
Comment (shakalee14) [inline docs/DB_MIGRATION.md:38]: agree

File: src/server/api/schema.js (hunk 1 above)
Comment (schuyler1d) [reply src/server/api/schema.js:370]: yes

**Comment by sandramchung** [comment]
👍

**Pull request merged**